### PR TITLE
create initial entry point for policy interaction in e2e tests

### PR DESCRIPTION
**Pull request by steeling**
create initial entry point for policy interaction in e2e tests

### CODE REVIEW

File: public.pgp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (qnuw) [inline public.pgp]: @steeling I don't think this should be committed.

File: tests/e2e/e2e_egress_validator_test.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package e2e
```
Comment (qnuw) [inline tests/e2e/e2e_egress_validator_test.go:1]: nit: whitespace looks off to me.

Comment (steeling) [reply tests/e2e/e2e_egress_validator_test.go:1]: removed the blank line if that's what you were referring to

Comment (qnuw) [reply tests/e2e/e2e_egress_validator_test.go:1]: I was referring to the indentation. It looks like more than 4 spaces?

**Review by cmgit — APPROVED** [review]
Will use this shell

**Pull request merged**